src/Runner.js

```diff
@@ -178,14 +204,23 @@ function run(transformFile, paths, options) {
     );
     return;
   } else {
-    return transform(transformFile);
+    return getStdin().then(str => transform(transformFile, str));
```

Comment:
You should check whether `process.stdin.isTTY` is false and only read from `process.stdin` in that case.